torchao/__init__.py

```diff
@@ -53,4 +53,5 @@
     "swizzle",
     "testing",
     "ops",
+    "quantization",
```

Comment:
why is swizzle top level?

Reply:
Thats a really good question

Reply:
but lets change in a different PR I could also remove this here